kjourney/src/app/app.module.ts

```diff
@@ -3,24 +3,36 @@ import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms'
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap'
 import { HttpClientModule } from '@angular/common/http'
+import { DiagramModule } from '@syncfusion/ej2-angular-diagrams';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { MainComponent } from './main/main.component';
 import { LoginComponent } from './login/login.component';
+import { ViewTabsComponent } from './view-tabs/view-tabs.component';
+import { View1Component } from './view1/view1.component';
+import { View2Component } from './view2/view2.component';
+import { View3Component } from './view3/view3.component';
+import { BandComponent } from './band/band.component';
 
 @NgModule({
   declarations: [
     AppComponent,
     MainComponent,
-    LoginComponent
+    LoginComponent,
+    ViewTabsComponent,
+    View1Component,
```

Comment:
what is going on here? what is the agreed view design you guys agreed on? remove all others!

Reply:
Removed all the other views and the tabs, leaving only the first view which will be rethought and reimplemented early next sprint